### ISSUE TITLE
Clicking element, a question between 2 different ways

### ISSUE THREAD

**Issue by mlipok** [Danp2/WebDriver#140]
in `wd_core.au3` we can use `_WD_ElementAction` to click with:

```AutoIt
Case 'clear', 'click'
	$sResponse = __WD_Post($_WD_BASE_URL & ":" & $_WD_PORT & "/session/" & $sSession & "/element/" & $sElement & "/" & $sCommand, '{"id":"' & $sElement & '"}')
	$iErr = @error
```

in `wd_helper.au3` we can use `_WD_ElementActionEx` with `DOUBLECLICK` :

which will be processed by:
```AutoIt
				$sResponse = __WD_Post($sURL, $sOption)
```

The question is whether, in your experience, WebDriver handles clicks differently in these two cases?
For example, are there problems with one case and not with another?

Thus can we add to `_WD_ElementActionEx` not only `DOUBLECLICK` but also `CLICK` ?
Or should we treat that there is already `CLICKANDHOLD` ?
Does it make sense?

**Comment by Danp2** [comment]
> The question is whether, in your experience, WebDriver handles clicks differently in these two cases?

IDK, but it's a good question. See [here](_WD_ElementActionEx ) for the Element Click specs. There may be cases where the "standard" click fails that could be resolved in _WD_ElementActionEx. One limitation of the default Click action is that it always clicks at the center of the element. There may be times where it would be desirable to control the location of the click within the element, so that would be something that could be added to _WD_ElementActionEx.

> Or should we treat that there is already CLICKANDHOLD ?

No, that's completely separate.

**Comment by Danp2** [comment]
Closed in cdc25199bb4c6073181f8ab8368c815559038574
